src/components/m-dropdown/_template.js

```diff
@@ -3,6 +3,12 @@ import raw from 'nanohtml/raw';
 import classnames from 'classnames';
 
 const arrowIcon = '<axa-icon size="sm" icon="angle-bracket-down" classes="m-dropdown__icon"></axa-icon>';
+const getSelectedItem = (items) => {
+  if (!items || !Array.isArray(items) || !items.length) {
+    return null;
+  }
+  return items.filter(item => item.isSelected)[0];
+};
```

Comment:
Generally, we should keep code "together" in the controller (index.js) instead of moving it into the view (like the `getSelectedItem` function). Now parts of code are in the template and others are in the index.js. A view should only act as view and should not prepare nor manipulate data ( and If we had tests, it would even be harder to test in the view).

In this specific example, The `selectedItem` is already existing on line 160 in index.js, it may be enough to pass this prop down to the template and access it from there, no need to do figure out the selected item with another helper `(getSElectedItem)` again.

Reply:
I agree. I'll update as soon as I can.

Reply:
I removed this helper and added selectedItem as attribute.